components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.tenant.initializer/src/main/java/org/wso2/carbon/apimgt/micro/gateway/tenant/initializer/internal/TenantInitializerServiceComponent.java

```diff
@@ -39,6 +39,9 @@
  * @scr.reference name="api.manager.config.service"
  * interface="org.wso2.carbon.apimgt.impl.APIManagerConfigurationService" cardinality="1..1"
  * policy="dynamic" bind="setAPIManagerConfigurationService" unbind="unsetAPIManagerConfigurationService"
+ * @scr.reference name="user.realm.service"
+ * interface="org.wso2.carbon.user.core.service.RealmService" cardinality="1..1"
+ * policy="dynamic" bind="setRealmService" unbind="unsetRealmService"
```

Comment:
why do we required this without any usages?

Reply:
It is used here 
https://github.com/amandaJayanetti/carbon-apimgt/blob/a192ecb8dde074704ab39481ed8b9922146423d8/components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.tenant.initializer/src/main/java/org/wso2/carbon/apimgt/micro/gateway/tenant/initializer/listener/ServerStartupListener.java#L90

Reply:
Ah okey I hope profiles not affected due to this :)